bundles/binding/org.openhab.binding.weather/src/main/java/org/openhab/binding/weather/internal/common/WeatherConfig.java

```diff
@@ -93,26 +103,32 @@ public void parse(Dictionary<String, ?> properties) throws ConfigurationExceptio
         for (LocationConfig lc : locationConfigs.values()) {
             if (!lc.isValid()) {
                 parseCompleted = true;
-                logger.warn("Incomplete location config for locationId '{}'. Check openhab.cfg.", lc.getLocationId());
+                logger.warn("Incomplete location config for locationId '{}'. Check openhab.cfg.",
+                    lc.getLocationId());
                 throw new ConfigurationException("weather",
-                        "Incomplete location config for locationId '" + lc.getLocationId() + "'. Check openhab.cfg.");
+                    "Incomplete location config for locationId '" +
+                    lc.getLocationId() + "'. Check openhab.cfg.");
```

Comment:
Please change all occurrences of `Check openhab.cfg` to something like `Check the binding config.`